Document-Processing-toc.html

```diff
@@ -147,31 +147,239 @@
+							</li>                 
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Magnification">Magnification</a></li>            
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Text-Search">Text Search</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Text-Selection">Text Selection</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Form-Filling">Form Filling</a></li>
+							<li>
+								<a href="/document-processing/pdf/pdf-viewer/maui/Annotations-Overview">Annotations</a>
+								<ul>
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Annotation-Collection">Annotation Collection</a></li>
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Add-Remove-Modify-Annotations">Add, Remove, and Edit</a></li>
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Select-Deselect-Annotations">Select and Deselect</a></li>	
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Lock-Unlock">Lock and Unlock</a></li>
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Import-Export-Annotations">Import and Export</a></li>
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Undo-Redo">Undo and Redo</a></li>
+									<li><a href="/document-processing/pdf/pdf-viewer/maui/Show-Hide">Show and Hide</a></li>
+									<li>
+										Types
+										<ul>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Ink">Ink</a></li>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Shapes">Shapes</a></li>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Stamps">Stamps</a></li>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Text-Markups">Text Markups</a></li>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Sticky-Notes">Sticky Notes</a></li>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Ink-Eraser">Ink Eraser</a></li>
+											<li><a href="/document-processing/pdf/pdf-viewer/maui/Free-Text">Free Text</a></li>
+										</ul>
+									</li>					
+								</ul>
+							</li>			
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Save-a-Document">Save a Document</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Print-a-Document">Print a Document</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Coordinates-Conversion">Coordinates Conversion</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Signature">Electronic Signatures</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Localization">Localization</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Right-To-Left">Right to Left</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Keyboard-Shortcuts">Keyboard Shortcuts</a></li>
+							<li><a href="/document-processing/pdf/pdf-viewer/maui/Gesture-Events">Gesture Events</a></li>
 						</ul>
 					</li>
```

Comment:
Please check the two files that has mentioned overview
<img width="359" height="179" alt="image" src="https://github.com/user-attachments/assets/0e138602-9f23-4b50-831a-b3959ce8ab27" />